src/distribution.cpp

```diff
@@ -301,9 +301,10 @@ void Tabular::init(
   }
 
   // Normalize density and distribution functions
+  integral_ = c_[n - 1];
```

Comment:
Same comment as before here. And in addition should we factor out the normalization methods and call them from within the constructor if that is needed?

Reply:
The response is pretty much the same as before. We're preserving the integral as the distribution is read in. Note that this is within the `init()` method which is indeed called from the constructor.